### ISSUE TITLE
Enable bots and make sure they are working correctly on stagnet 3

### ISSUE THREAD

**Issue by gordsport** [vegaprotocol/vegacapsule#161]
Enable bots and make sure they are working correctly on stagnet 3 

For now we should use the existing bots

Info for later:
Later on Aleksander will be working on improving the bots to work with capsule - a new issue will be raised to use in infra when ready?
- https://github.com/vegaprotocol/liqbot/issues/142

**Comment by gordsport** [comment]
blocked by:
- https://github.com/vegaprotocol/vegacapsule/issues/160

**Comment by gordsport** [comment]
Closing as in `Done` column
